src/global.ts

```diff
@@ -186,9 +188,9 @@ interface GetTokenSilentlyOptions extends GetUserOptions {
    */
   [key: string]: any;
 }
-interface GetTokenWithPopupOptions extends PopupLoginOptions {}
+export interface GetTokenWithPopupOptions extends PopupLoginOptions {}
```

Comment:
We should probably put a comment on this to remove it at the next major release.

Reply:
Done